### DISCUSSION TITLE
Unable to start install process

### DISCUSSION THREAD

**Discussion by MaryMuse** [bookwyrm-social/bookwyrm#2438] [Q&A]
I created a docker droplet from DO from the marketplace and when I went to run the first instruction here: https://docs.joinbookwyrm.com/install-prod.html

git clone git@github.com:bookwyrm-social/bookwyrm.git

I get this error message:
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

I've tried creating my droplet both with a root password and a ssh key.

Anyone assist? Thanks.

**Comment by mouse-reeve** [comment]
I'm not sure what the correct fix is, but cloning via https instead of ssh should work

**Reply by dglloyd** [reply]
Generally this means the user you are cloning as does not have an ssh key configured with github. https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account should get you started.
